File: 0268-missing-number/0268-missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
 function missingNumber(nums){
     // let ans = n;
    for(let i=0;i<=nums.length;i++){
        if(!nums.includes(i)){
            return i
        }
    }
 }

File: 0268-missing-number/0268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
     vector<int> ans;
      int n = nums.size();
      sort(begin(nums),end(nums));
      for(int i=0;i<n;i++){
        if(nums[i] != i){
          return i;
        }
      }
      return n;
    }
};